media/css/firefox/family/components/_hero.scss

```diff
@@ -39,8 +39,52 @@ $image-path: '/media/protocol/img';
     }
 
     .c-blurb {
-        background-color: f3.$green-light;
+        @include text-body-lg;
+        @include font-firefox;
+
+        --shadow-offset: #{$border-radius-md};
+        @include f3.card-shadow;
+        background-color: f3.$blue-light;
+        border-radius: $border-radius-lg;
+        color: inherit;
+        display: inline-block;
+        font-weight: bold;
         margin-top: $layout-lg;
+        text-align: center;
+        text-decoration: none;
+
+        @media (prefers-reduced-motion: no-preference) {
```

Comment:
note: there might be some design changes here for the mobile behaviour of the CTA link

Reply:
resolved to reduce the scale effect on mobile, we still see an outline on long taps but that appears to be mobile browser default behaviour and we are considering it non-blocking